V2/tezex/src/theme.tsx

```diff
@@ -45,11 +45,11 @@ const theme = createTheme({
 });
 /*
 const theme: Theme = createTheme({
-	palette: {
-		grey: {
+  palette: {
+    grey: {
       main: purple[500],
-		}
-	},
+    }
+  },
   typography: {
     fontFamily: "Inter",
    },
```

Comment:
Best to delete old code.

Reply:
Will do , its still a draft started work on it today .

Reply:
Yes, realized it's in draft only after submitting the "review". :clown_face: